### PR TITLE
LPS-94573 polls-test: Create PollsQuestionMultiLanguageSearchTest

### PR DESCRIPTION
**Pull request by brandizzi**

### CODE REVIEW

**Comment by foshiro-zz** [comment]
ci:test:sf

**Comment by foshiro-zz** [comment]
ci:test:search

File: modules/apps/polls/polls-test/src/testIntegration/java/com/liferay/polls/search/test/PollsQuestionMultiLanguageSearchTest.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+import com.liferay.portal.kernel.search.Document;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
+import com.liferay.portal.kernel.test.rule.SynchronousDestinationTestRule;
+import com.liferay.portal.kernel.util.LocaleThreadLocal;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.search.test.util.FieldValuesAssert;
+import com.liferay.portal.search.test.util.IndexerFixture;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerTestRule;
```
Comment (foshiro-zz) [inline modules/apps/polls/polls-test/src/testIntegration/java/com/liferay/polls/search/test/PollsQuestionMultiLanguageSearchTest.java]: Please, change to PermissionCheckerMethodTestRule

File: modules/apps/polls/polls-test/src/testIntegration/java/com/liferay/polls/search/test/PollsQuestionMultiLanguageSearchTest.java [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+ */
+@RunWith(Arquillian.class)
+public class PollsQuestionMultiLanguageSearchTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerTestRule.INSTANCE,
```
Comment (foshiro-zz) [inline modules/apps/polls/polls-test/src/testIntegration/java/com/liferay/polls/search/test/PollsQuestionMultiLanguageSearchTest.java]: Please, change to PermissionCheckerMethodTestRule

**Review by foshiro-zz — CHANGES_REQUESTED** [review]
@vagner-castro , code and test looks good to me. Just have to change to PermissionCheckerMethodTestRule

**Comment by vagnerbc** [comment]
@foshiro Changes updated

**Review by foshiro-zz — APPROVED** [review]

**Comment by foshiro-zz** [comment]
Submitted to https://github.com/brandizzi/liferay-portal/pull/782

**Pull request closed without merging**